src/api/map/routes/map.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable require-await */
+import { procedure, router } from "../../trpc";
+import { authUser } from "../../auth";
+import { z } from "zod";
+import { zRoles } from "shared/Role";
+
+const zLandmark = z.object({
+  定义: z.string(),
+  经度: z.string(),
+  纬度: z.string(),
+  层级: z.array(z.string()),
+  相关地标: z.array(z.string()),
+  工具箱: z.string(),
+});
+
+//type Landmark = z.TypeOf<typeof zLandmark>;
+
+const list = procedure
+  .use(authUser())
+  .input(z.object({
```

Comment:
the input doesn't need any of these fields. it instead needs lattitude which should be an enum.